scene/3d/light_3d.cpp

```diff
@@ -165,6 +165,16 @@ AABB Light3D::get_aabb() const {
 	return AABB();
 }
 
+PackedStringArray Light3D::get_configuration_warnings() const {
+	PackedStringArray warnings = Node::get_configuration_warnings();
```

Comment:
Should calling direct parent version, otherwise you're bypassing potential `VisualInstance3D`, etc. warnings.